src/variableProcessor.ts

```diff
@@ -48,6 +53,23 @@ export class VariableProcessor {
             }
         }
 
+        let requestVariables = VariableProcessor.getRequestVariablesInCurrentFile();
+        for (let [variableName, variableValue] of requestVariables) {
+            let regex = new RegExp(`\\{\\{\\s*${variableName}.*\\s*\\}\\}`, 'g');
+            let matches = request.match(regex);
+            if (matches && matches.length > 0) {
+                for (let i = 0; i < matches.length; i++) {
+                    const requestVariable = matches[i].replace('{{', '').replace('}}', '');

```

Comment:
should also remove whitespaces after `{{` and before `}}`. So it should be something like
```js
const requestVariable = match[i].replace(/\{\{\s*/, '').replace(/\s*\}\}/, '');
```

Reply:
Is it a good idea to allow whitespace after `{{` and before `}}` in variables? Is it a general requirement?

Then I cannot re-use utilities like `isVariableReferenceFromLine` in `variableUtility.ts`. In fact that will require a bit of re-write to support hover & completion.

Reply:
Agree, keep this logic.